imap_processing/mag/l0/decom_mag.py

```diff
@@ -40,11 +46,14 @@ def decom_packets(packet_file_path: str) -> list[MagL0]:
 
     with open(packet_file_path, "rb") as binary_data:
         try:
-            mag_packets = mag_parser.generator(binary_data)
+            mag_packets = mag_parser.generator(
+                binary_data,
+                buffer_read_size_bytes=5790778,  # Todo: what size?
+            )
```

Comment:
Speaking with Gavin earlier in that thread on slack we shouldn't be updating this, and in fact it makes it much slower due to the buffer thrashing issues on space_packet_parser. We can safely use the default.
```suggestion
            mag_packets = mag_parser.generator(binary_data)
```

Reply:
Yup, just missed updating it here, I will remove!

Reply:
Awesome, thanks!